Changelog.md

```diff
@@ -1,5 +1,19 @@
 ## Ignition Rendering
 
+### Gazebo Rendering 3.7.2 (2024-01-05)
+
+1. Update github action workflows
+    * [Pull request #940](https://github.com/gazebosim/gz-rendering/pull/940)
+
+1. Add minor comments to BaseGizmoVisual
+    * [Pull request #881](https://github.com/gazebosim/gz-rendering/pull/881)
+
+1. CI workflow: use checkout v3
+    * [Pull request #834](https://github.com/gazebosim/gz-rendering/pull/834)
```

Comment:
As @azeey commented elsewhere, this can be consolidated with the other github housekeeping entries.

Reply:
done. [a371e06](https://github.com/gazebosim/gz-rendering/pull/952/commits/a371e06d2bb1ae1c8ba8cb920eb9f7a2e66a5acf)